src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If you doing last few homeworks as a separate project - you should have only one pom.xml, one application.yml, one log4j.xml, one start point for spring boot app